doc/source/parameters/cfd/laser_heat_source.rst

```diff
@@ -30,44 +31,74 @@ If a laser heat source is present in a simulation, it can be added in this secti
+
+  .. attention::
+    This parameter is only taken into account if the laser ``type`` is set to ``exponential_decay``.
 
 * The ``beam radius`` parameter defines the radius of the laser beam.
 
 * The ``start time`` and ``end time`` parameters define the operation time window of the laser.
 
-* The ``beam orientation`` parameter shows the orientation and direction of the laser beam. For instance, if a laser beam is emitted perpendicular to a plane in x-y coordinates, the orientation of the laser beam will be in the z-direction. Negative (-) or positive (+) defines the direction of the laser beam. For instance if the laser beam is emitted in the negative z direction, the value of ``beam orientation`` will be ``z-``.
+* The ``beam orientation`` parameter shows the orientation and direction of the laser beam. For instance, if a laser beam is emitted perpendicular to a plane in :math:`x`-:math:`y` coordinates, the orientation of the laser beam will be in the z-direction. Negative (-) or positive (+) defines the direction of the laser beam. For instance if the laser beam is emitted in the negative :math:`z` direction, the value of ``beam orientation`` will be ``z-``.
 
-.. note::
-    In two-dimensional simulations, the laser beam orientation cannot be in the z-direction.
+  .. attention::
+      In two-dimensional simulations, the laser beam orientation cannot be in the z-direction.
 
 
 * In the ``path`` subsection, the laser scanning path is defined using a ``Function expression``.
 
-* ``subsection free surface radiation``: In additive manufacturing simulations, radiation at the interface between the air and the metal is a significant cooling mechanism. When this interface (i.e., free surface) is resolved by the :doc:`volume_of_fluid` solver, the ``free surface radiation`` subsection defines the parameters to impose this radiation cooling following the Stefan-Boltzmann law of radiation: :math:`\epsilon \sigma (T^4 - T_{inf}^4)`.
+* ``subsection free surface radiation``: In additive manufacturing simulations, radiation at the interface between the air and the metal is a significant cooling mechanism. When this interface (i.e., free surface) is resolved by the :doc:`volume_of_fluid` solver, the ``free surface radiation`` subsection defines the parameters to impose this radiation cooling following the Stefan-Boltzmann law of radiation:
+
+  .. math::
+      q_\text{rad} = \epsilon \sigma (T^4 - T_\text{inf}^4)
 
-  * ``enable``: controls if the radiation cooling is enabled. The radiation sink is apply in the heat transfer solver in the cells where the phase fraction gradient norm :math:`|\nabla \alpha|` is non-null.
+  * ``enable``: controls if the radiation cooling is enabled. The radiation sink is applied in the heat transfer solver in the cells where the filtered phase fraction gradient norm :math:`|\nabla \phi'|` is non-null.
```

Comment:
I saw the equation and the code, so I understand better what you are saying here. But instead of saying that it is applied in the cells where the phase gradient is non-0, I would say that the norm of the phase gradient weights the sink term since you solve this on all cells anyway.